kitsune/settings.py

```diff
@@ -625,6 +625,7 @@ def JINJA_CONFIG():
             ('**/templates/**.ltxt', 'jinja2'),
             ('kitsune/**.py', 'python'),
             ('kitsune/**/templates/**.html', 'jinja2'),
```

Comment:
I think we have a couple templates that override Django templates that this would catch. I'm not sure if they use localization though?